1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
+menu_order: 1
+---
+
+The cluster address may be obtained using the following methods:
```

Comment:
I indeed wish that we could settle on "cluster base URL" or "cluster URL". This concept and term will hopefully become a first class citizen. In URL terminology, it's meant to include scheme, network location, and no path. It's the _intended_ way to access the cluster.